test/resources/test-flavor/flavor_base

```diff
@@ -0,0 +1 @@
+real_flavor_base/
```

Comment:
so this is still in the same flavor root directory, or? I thought, you had created a symlink on the flavor root directory, to test if your symlink resolution works. With this you only test, if relative symlinks in the same parent directory work, which they should. However, as a negative test, we could create another flavor which then has a symlink for flavor_base. This probably, would fail with the docker-starter, except both directory are under another mounted directory.